p256/src/arithmetic/scalar.rs

```diff
@@ -130,10 +130,28 @@ impl Scalar {
     }
 
     /// Returns the multiplicative inverse of self, if self is non-zero
+    #[cfg(not(target_os = "zkvm"))]
     pub fn invert(&self) -> CtOption<Self> {
         CtOption::new(self.invert_unchecked(), !self.is_zero())
     }
 
+    /// Returns the multiplicative inverse of self, if self is non-zero
+    #[cfg(target_os = "zkvm")]
+    pub fn invert(&self) -> CtOption<Self> {
+         // NOTE: we dont care about constant time operations in the vm.
```

Comment:
Please use proper comments here with proper punctuation and no first-person tense.